src/fmgc/src/guidance/lnav/LnavDriver.ts

```diff
@@ -36,7 +36,7 @@ export class LnavDriver implements GuidanceComponent {
         console.log('[FMGC/Guidance] LnavDriver initialized!');
     }
 
-    update(_deltaTime: number): void {
+    update(_: number): void {
```

Comment:
Not a fan of this change tbh, as now I don't know what that param is if I do want to use it in future. It would be better if the lint rules didn't make us add the underscore at all, as it really adds no value, only diff/churn, especially with a modern editor that would highlight the unused param appropriately.

Reply:
I only picked this solution if the function was implemented based on an interface, in other cases I removed the parameter. Personally I'd have picked removal of the parameter full stop. We cannot keep the parameter in with a regular name and leave it unused, unless we disable the unused parameter rule.

What do you suggest?

Reply:
There's no easy + good solution, so let's continue with the change as-is I guess. 😛